samples/generated/v2/snippet_metadata.google.bigtable.v2.json

```diff
@@ -0,0 +1,327 @@
+{
+  "clientLibrary": {
+    "name": "nodejs-bigtable",
+    "version": "0.1.0",
```

Comment:
@sofisl could you take a look? Is the version right to you?

Reply:
This should be the current version of the library, but there's no way to put that information into the generator. I think, for now, we should just change it manually, but then have release-please update this value when a new release is made. I'll open a bug on release-please.